src/solver/conv_asm_implicit_gemm_gtc_perf_config.cpp

```diff
@@ -88,13 +88,70 @@ PerformanceConfigAsmImplicitGemmGTC::PerformanceConfigAsmImplicitGemmGTC(
 {
 }
 
+PerformanceConfigAsmImplicitGemmGTC::PerformanceConfigAsmImplicitGemmGTC(
+    std::string dir,
+    std::string layout,
+    miopenDataType_t prec,
```

Comment:
We need to maintain ctor where precision is miopenDataType_t only to initialize vectors ("lists") in Solvers.